lib/screen/interactive_postcard/postcard_detail_bloc.dart

```diff
@@ -55,6 +56,7 @@ class PostcardDetailBloc
     this._indexerService,
   ) : super(PostcardDetailState(provenances: [])) {
     on<PostcardDetailGetInfoEvent>((event, emit) async {
+      await LocaleService.getLocaleMeasurement();
```

Comment:
it can be reduced to only refresh the data when app restore from background.

see `_handleForeground` in homepage.